### ISSUE TITLE
bug: CSS issues or UI bugs on Android 10 devices?

### ISSUE THREAD

**Issue by petrot** [ionic-team/ionic-framework#22372]
#22321 Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**
3-4 weeks ago, my Ionic 5 (angular) app started to showing weird UI bugs: missing button labels, empty popovers, etc..
The problem is appeared only on Android devices with Android 10. Has anyone met anyone like that? It could be a webview bug? Or Ionic specific?

**Expected behavior:**
Display correctly.

**Steps to reproduce:**
I can't reproduce, on virtual Android 10 device there are no problems.

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you reproduce this issue in a blank Ionic starter application? We are not aware of any outstanding issues specifically on Android 10.

**Comment by ThonyFD** [comment]
@petrot it's something like this? we are facing some weird view issues on our Ionic 4 app in production and it starts 3-4 weeks ago. The top reporter is Huawei/Android 10.

![Screenshot_20201009_104518_com android settings (2)](https://user-images.githubusercontent.com/1325872/96889161-f296bd80-144b-11eb-8893-9073b5d351bd.jpg)
![IMG-20201009-WA0021](https://user-images.githubusercontent.com/1325872/96889173-f591ae00-144b-11eb-9129-0984087a6816.jpg)

**Comment by liamdebeasi** [comment]
It's hard to say what the issue is without being able to reproduce the issue myself. What is the result of logging `window.navigator.userAgent` in the console when debugging the app?

**Comment by petrot** [comment]
@ThonyFD yes! Mainly the Samsung and Huawei devices are affected. But an other leaflet marker error appeared on a Xiamoi device (Android 10, too)

@liamdebeasi it's very hard to reproduce.. I have Apple devices and 2 Android phones with Android 9. No error appears on them. With Android 10 emulator - no problem...

Some other screen:

![image](https://user-images.githubusercontent.com/4799473/96912357-8c895500-14a2-11eb-9621-aae39ce9cd81.png)
![image](https://user-images.githubusercontent.com/4799473/96912370-90b57280-14a2-11eb-9a37-2bbe12710b35.png)
![image](https://user-images.githubusercontent.com/4799473/96912381-93b06300-14a2-11eb-8fc3-ed1dfbe64d52.png)

**Comment by liamdebeasi** [comment]
That seems more like a webview/device rendering issue, not something specific to Ionic Framework. Unfortunately I cannot reproduce this on my end so if someone is able to provide steps to reproduce that would be helpful.

**Comment by liamdebeasi** [comment]
Hi everyone,

I did a quick search on the Chromium bug reporter and it turns out this is a bug in Chromium: https://bugs.chromium.org/p/chromium/issues/detail?id=1136975&sort=-modified&q=os%3DAndroid%20artifact&can=2

Here is a breakdown of the issue:

1. This visual glitch was caused by the enabling of the "VizForWebView" Chromium experiment starting around Chrome 85.
2. The experiment has been disabled.
3. The issue should disappear whenever the WebView gets a new experiments config update.

Thanks!
